object_detection/scripts/get_objects_and_coordinates.py

```diff
@@ -0,0 +1,136 @@
+"""
+This script test the trained model and performs object detection on a given image, the script prints
+in console a JSON with the objects detected and their info.
+
+E.g. 
+>>> python get_objects_and_coordinates.py -d test_images/image1.jpg -i false
+output:
+{
+    "jumex_cajita": {
+        "score": 0.9995954632759094,
+        "ymin": 435.3594779968262,
+        "xmin": 185.599547624588,
+        "ymax": 547.5826263427734,
+        "xmax": 248.1843888759613
+    },
+    "fanta": {
+        "score": 0.9212740063667297,
+        "ymin": 314.3073320388794,
+        "xmin": 328.5956025123596,
+        "ymax": 473.17280769348145,
+        "xmax": 405.4909944534302
+    }
+}
+"""
+
+import os
+import argparse
+import cv2
+import numpy as np
+import tensorflow as tf
+import sys
+import json
+
+from utils import label_map_util
+from utils import visualization_utils as vis_util
+
+from google.colab.patches import cv2_imshow
+
+def run_model(image_directory):
+    """
+    This function loads the neccesary files, runs the model and gets the detection boxes, scores and classes.
+    """
+    global boxes, scores, classes, num, category_index, image
+
+    MODEL_NAME = 'inference_graph'
+    IMAGE_NAME = image_directory
+    CWD_PATH = os.getcwd()
+    PATH_TO_CKPT = os.path.join(CWD_PATH,MODEL_NAME,'frozen_inference_graph.pb')
+    PATH_TO_LABELS = os.path.join(CWD_PATH,'training','labelmap.pbtxt')
```

Comment:
Maybe later we can make here a checking of the existence of these and say where to get it. As an example check [how is done in DeepSpeech](https://github.com/RoBorregos/Robocup-Home/blob/develop/catkin_home/src/action_selectors/scripts/DeepSpeech/deploy/ros_server.py#L163-L222).

Reply:
Yes, its a good idea